components/ui/InfiniteScroll.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <section>
+    <div
+      v-for="(item, index) in items.slice(0, lastShown)"
+      :id="index + '-' + keyGenerator(item)"
+      :key="keyGenerator(item)"
```

Comment:
I believe we can simplify this like so:

```suggestion
      :id="`${_uid}-${index}`"
      :key="index"
```

Since the elements in the array used by the `v-for` don't change order, I think that the `index` of the list should a good **key** here.

In order to avoid duplicate IDs, we could use Vue's component `_uid` and then append the index.

Reply:
The elements will be filtered at some point. I think that invalidates the key.

Reply:
The elements will be filtered at some point. I think that invalidates the key, but I'm not sure